public/index.html

```diff
@@ -56,9 +56,19 @@ <h2>
       </div>
     </footer>
     <script src="./assets/javascripts/application.js?v=1.4" type="text/javascript"></script>
-    <script type="text/javascript">
-      var url = new URL(window.location.href);
+	<script type="text/javascript">
+   	var url = new URL(window.location.href);
       document.getElementById('receiver').value = url.searchParams.get("address");
     </script>
+	 <script type="text/javascript">
+		document.getElementById("requestTokens").addEventListener("click", tempDisableReCAPTCHA);
+
+		function tempDisableReCAPTCHA() {
+    		document.getElementById('requestTokens').disabled = true;
```

Comment:
@InternetJohnny let's increase readability allocating `document.getElementById('requestTokens')` into the varaible and use it here and below.